crates/stellar-scaffold-cli/src/commands/build/clients.rs

```diff
@@ -358,18 +366,40 @@ export default new Client.Client({{
 
         for account in accounts {
             eprintln!("🔐 creating keys for {:?}", account.name);
-            cli::keys::generate::Cmd::parse_arg_vec(&[&account.name, "--fund"])?
-                .run(&stellar_cli::commands::global::Args::default())
+            let args = stellar_cli::commands::global::Args {
+                locator: self.clone().get_config_locator(),
+                ..Default::default()
+            };
+
+            match cli::keys::generate::Cmd::parse_arg_vec(&[&account.name, "--fund"])?
+                .run(&args)
                 .await
-                .or_else(|e| {
-                    if e.to_string().contains("already exists") {
-                        // ignore "already exists" errors
-                        eprintln!("{e}");
-                        Ok(())
-                    } else {
-                        Err(e)
+            {
+                Ok(()) => {}
+                Err(e) if e.to_string().contains("already exists") => {
```

Comment:
I was thinking we would only fund if we got an "account doesn't exist" situation.

A flow like:
- create the keys, ignore already exists errors
- check that the account exists
- account doesn't exist? fund on not-mainnet

The approach here _always_ funds if the account currently exists, which would mean that on many random days (since you can fund 1x day) you'd fund your account when you start your app. This seems weird, and undesirable.